src/main/java/org/zalando/nakadi/repository/kafka/KafkaLocationManager.java

```diff
@@ -1,101 +1,21 @@
 package org.zalando.nakadi.repository.kafka;
 
-import org.apache.curator.framework.CuratorFramework;
-import org.apache.kafka.clients.CommonClientConfigs;
 import org.apache.kafka.clients.consumer.ConsumerConfig;
 import org.apache.kafka.clients.producer.ProducerConfig;
-import org.json.JSONException;
-import org.json.JSONObject;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.scheduling.annotation.Scheduled;
-import org.zalando.nakadi.repository.zookeeper.ZooKeeperHolder;
 
-import java.io.UnsupportedEncodingException;
-import java.util.ArrayList;
-import java.util.List;
 import java.util.Properties;
 
 public class KafkaLocationManager {
```

Comment:
The remaining methods in this class could probably be moved to `KafkaFactory` and this class removed.

Reply:
Just tell me if I do it.